test/e2e/specs/site-editor/user-global-styles-revisions.spec.js

```diff
@@ -29,10 +22,9 @@ test.describe( 'Global styles revisions', () => {
 		await requestUtils.activateTheme( 'twentytwentyone' );
 	} );
 
-	test.beforeEach( async ( { admin } ) => {
-		await admin.visitSiteEditor( {
-			canvas: 'edit',
-		} );
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.visitSiteEditor();
+		await editor.canvas.click( 'body' );
```

Comment:
@ramonjd, I wasn't able to run tests locally without this change. However, it's a common pattern, so there's no need for a separate PR.

Reply:
No worries, thanks for the heads up!